src/main/resources/i10n/AribaBatchSourceBundle.properties

```diff
@@ -0,0 +1,10 @@
+## Mandatory UI parameter validation error messages
+err.missing.param.prefix=Required property ''{0}'' is blank.
```

Comment:
Why is the placeholder inside two single quotes? Why not just one single quote?

Reply:
Two single quote is added so that when error comes on the UI the property will be visible in single quote. User will able to see the property name more clearly. E.g.  Required property 'API Endpoint' is blank.